app/vmalert/notifier/alert.go

```diff
@@ -187,7 +187,8 @@ func templateAnnotation(dst io.Writer, text string, data tplData, tmpl *textTpl.
 	return nil
 }
 
-func (a Alert) toPromLabels(relabelCfg *promrelabel.ParsedConfigs) []prompbmarshal.Label {
+// applyRelabeling applies relabeling rules to the alert labels before sending it.
+func (a Alert) applyRelabeling(relabelCfg *promrelabel.ParsedConfigs) map[string]string {
```

Comment:
`apply` prefix assumes the object will be mutated. Maybe it could have a better name

Reply:
Updated to `applyRelabelingIfNeeded`.